ranking/views.py

```diff
@@ -13,3 +14,42 @@ class RankingView(LoginRequiredMixin, SingleTableView):
     def get_queryset(self):
         return ChallengeUser.objects.filter(success=True).values('user__username')\
             .annotate(count=Count('*'), time=Max('last_try')).order_by('-count', 'time')
+
+
+def get_chart(request):
+    # Get all Successful attempts
+    allEntries = list(ChallengeUser.objects.filter(success=True).values('user__username', 'last_try', 'total_attempts')
+                      .order_by('last_try'))
+
+    players = {}
+    Pcounter = {}
```

Comment:
Variables don't usually start in uppercase. In Phthon only classes start with uppercase.

Reply:
Fixed, thanks 4 the reminder